packages/admin/tsconfig.json

```diff
@@ -1,8 +1,21 @@
 {
   "extends": "../../tsconfig.json",
-  "compilerOptions": { "outDir": "./dist/cjs", "rootDir": "./src" },
+  "compilerOptions": {
+    "outDir": "./dist/cjs",
+    "rootDir": "./src"
+  },
   "references": [
-    { "path": "../grpc-authentication" },
-    { "path": "../grpc-connection" }
+    {
```

Comment:
all sub-packages have now been updated with their references, so all local references should be up to date